src/main/java/com/homework/springboot/service/UserRepositoryService.java

```diff
@@ -0,0 +1,15 @@
+package com.homework.springboot.service;
+
+import com.homework.springboot.model.User;
+
+import java.util.List;
+
+public interface UserRepositoryService {
```

Comment:
Is it a repository or service.
I believe that it is a service. If I am right please rename it.

Reply:
You should rename this class! You did not answer or change the code, just marked them as resolved.
Please add an explanation.